glustercli/cmd/volume-create.go

```diff
@@ -102,10 +104,16 @@ func smartVolumeCreate(cmd *cobra.Command, args []string) {
 		failure("Invalid File Size specified", nil, 1)
 	}
 
+	maxBrickSize, err := sizeToBytes(flagCreateMaxBrickSize)
```

Comment:
if `flagCreateMaxBrickSize` has changed then do size calculation?

Reply:
sizeToBytes returns zero when input is empty, no conversion/calculation done